### DISCUSSION TITLE
Swimmers Names Not Showing w/ SCB files and Correct Directory

### DISCUSSION THREAD

**Discussion by scstanton20** [JohnStrunk/wahoo-results#138] [Q&A] [answered]
First of all, thank you so much for all the work you have done! My club installed a dedicated outdoor TV to use this software during meets, and its basic functionality has been great so far! Unfortunately, I am having a problem displaying the swimmer's names and event name. I have correctly exported the .SCB files from meet manager, and have also correctly told Wahoo where the folder is. Please confirm that MM has correctly exported the files using the attached examples, and look at my screenshot to make sure I haven't skipped anything.

I'm on version 0.8.0.
![Screenshot 2022-07-16 094202](https://user-images.githubusercontent.com/98724625/179357477-7582d687-2889-4dfa-900b-f3f24d5bbcbf.png)
![Screenshot 2022-07-16 094225](https://user-images.githubusercontent.com/98724625/179357480-162ea056-e77d-446e-881c-a2b1f9093206.png)
![Screenshot 2022-07-16 094305](https://user-images.githubusercontent.com/98724625/179357481-3efcd5f7-5db9-40bd-8dd1-019ca3c32a4c.png)

**Comment by dmanusrex** [comment]
Did you export the dolphin events from wahoo resukts and import them into dolphin? Your event/heat in dolphin needs to match so it can find the correct data.  

https://wahoo-results.readthedocs.io/en/stable/configuration.html

**Reply by scstanton20** [reply]
Yep, used the .csv from wahoo to import into CTS Dolphin.
![Screenshot 2022-07-16 102022](https://user-images.githubusercontent.com/98724625/179358820-1237a2f9-0883-42b2-995d-895ac88dc434.png)

**Comment by dmanusrex** [comment]
On your preview pane it showed event 33 heat 1.  Do you have an E033 startlist file?

**Reply by scstanton20** [reply]
Yes, see attached.
![e033](https://user-images.githubusercontent.com/98724625/179359604-84d18f59-95ef-4cee-862a-1f9ece0af9e3.png)

**Comment by dmanusrex** [comment]
Doing this from memory… you’ll have to wait for John to chime in.  I’m sure i am forgetting something simple.

**Comment by JohnStrunk** [comment]
I'm glad you're finding it useful! It was a big hit w/ my daughter's team to have a scoreboard for our meets.

It looks like @dmanusrex hit all the configuration items... Thanks for jumping in.

I think this is actually a bug in the scoreboard. We always start our events at number 101 (3 digits to the event number). It looks like I'm not properly constructing the SCB filename properly for smaller event numbers. I think it's looking for `E33.scb` instead of `E033.scb`. I'll confirm and put a fix together in the next couple days.

**Reply by scstanton20** [reply]
Thank you! That makes sense- our events typically number from 1-69. With the summer season coming to an end, we are hosting the divisonals meet in 2 weeks and would love to show off our new scoreboard. Thanks!

**Reply by JohnStrunk** [reply, marked as answer]
I've pushed a new release [0.8.1](https://github.com/JohnStrunk/wahoo-results/releases/download/v0.8.1/wahoo-results.exe) that should fix your issue. Please give it a try and let me know.

Since the scoreboard is triggered by the `d04` files appearing in the directory, an easy way to test is to move them to another directory then move them back one at a time.
The names should also properly appear in the "manual results" screen (press the "Load" button on the bottom).

**Reply by scstanton20** [reply]
Thanks for pushing an update so fast! This resolved my issue.

**Reply by JohnStrunk** [reply]
@Scstanton20 I'd like to be able to test new releases against additional data (other than what I have from my meets). I'd really appreciate if you could send me a set of d04 files from one of your meets. Since they just have times and no other info, I don't think there's anything sensitive in there.
I can generate corresponding start lists with fake names.
If you're on-board, you can mail them to me at john.strunk at gmail. If you're not comfortable sending me the data, that's fine too.
